indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -331,7 +328,18 @@ public static boolean runJobs(List<Jobby> jobs, HadoopDruidIndexerConfig config)
         Path workingPath = config.makeIntermediatePath();
         log.info("Deleting path[%s]", workingPath);
         try {
-          workingPath.getFileSystem(injectSystemProperties(new Configuration())).delete(workingPath, true);
+          if( workingPath.getFileSystem(injectSystemProperties(new Configuration())).delete(workingPath, true) ){
```

Comment:
@fjy do you mean line too long? Maybe I should add a local variable to hold the "FileSystem" object before this "if" statement.